utils/reconciler.go

```diff
@@ -448,7 +469,7 @@ func (r *ReconcilerBase) GenerateSvcCertSecret(ba common.BaseComponent, prefix s
 		bao := ba.(metav1.Object)
 
 		err = r.GenerateCMIssuer(bao.GetNamespace(), prefix, CACommonName, operatorName)
```

Comment:
Should this be skipped if a custom issuer exists?

Reply:
I decided to leave this on just so that it is always there,  and since default issuer / CA are not owned by application they will not be deleted with apps. 
We might want to revisit in the future, just didn't want to add complexity of cleanup at this point

Reply:
Okay, let's revisit. Agree that we don't need to clean up the default issuer and its resources, but the method call GenerateCMIssuer can be skipped if a custom issuer is used.